src/ajitSJune20/CountLetDigSpclChar.java

```diff
@@ -0,0 +1,34 @@
+//Count Digits,Letters, UpperCase characters, LowerCase characters, Special characters from predefined string.
+
+package ajitSJune20;
+
+public class CountLetDigSpclChar {
+	
+	void displayCount(String str) {
+		int upper = 0, lower = 0, digit = 0, special = 0; 
+		
+		for(int index=0;index<str.length();index++) {
+			char ch=str.charAt(index);
+			if(Character.isDigit(ch))
+				digit++;
+			else if(Character.isUpperCase(ch))
+				upper++;
+			else if(Character.isLowerCase(ch))
+				lower++;
+			else
+				special++;
+		}
+		/*System.out.println("Total Digit:--------> "+digit);
+		System.out.println("Total UpperCase:----> "+upper);
+		System.out.println("Total LowerCase:----> "+lower);
+		System.out.println("Total SpecialChar:--> "+special); */
+		System.out.println("Total Digit:--------> "+digit + "\nTotal UpperCase:----> "+upper+"\nTotal LowerCase:----> "+lower+"\nTotal SpecialChar:--> "+special);
```

Comment:
Ajit, here you forget to print total letters count.

Reply:
Incorporated Changes.

Reply:
![image](https://user-images.githubusercontent.com/66831507/85765807-16fbbb00-b734-11ea-9720-68ffc651af01.png)